packages/ag-charts-community/src/scene/shape/shape.ts

```diff
@@ -188,7 +188,7 @@ export abstract class Shape<TDatum = unknown> extends Node<TDatum> {
     }
 
     protected fillStroke(ctx: CanvasContext, path?: Path2D) {
-        if (this.drawingMode === 'cutout') {
+        if ((this as any).__drawingMode === 'cutout') {
```

Comment:
These `as any` casts seem a little sketchy, losing quite a bit of type safety in shape, rect and barShape.